### DISCUSSION TITLE
How invoke BLEOp over MQTT

### DISCUSSION THREAD

**Discussion by BlackOrca** [arendst/Tasmota#21662] [Q&A] [answered]
Hi everybody,
at first Thanks for this awesome peace of software!

For my project i use the Bluetooth functionality on a esp32. I flashed tasmota32.bluetooth 14.1.0.
When i invoke "BLEOp M:c000000XXXXX s:FF01 c:FF02 r go" in the console i got that what i want.

But now my question, how can i invoke this with mqtt.
When i publish this as it to 
"cmnd/name/BLEOp M:c000000XXXXX s:FF01 c:FF02 r go"  
i got unknown command in the console of the tasmota.

The Documentation show only the Toggle command. Or Power 1, Power 0 
When i use only BLEOp i got in console that the command is prepared so i can invoke BLEOp2 but in my fist request i have no subscription etc.

Can anybody switch the light on in my head?

Br,
BlackOrca

**Comment by sfromis** [comment, marked as answer]
When using MQTT, you need to separate the command into the topic field (which is the "address" you publish to) and the payload field (with the argument values). This is after the first space in what you tried to do.

**Reply by BlackOrca** [reply]
Hi @sfromis,
Many thanks. My fault was to try to get this arguments into a Json payload. But now i have tried with a plaintext string and thats works great!
Many thanks!
